src/neynar-api/v2/client.ts

```diff
@@ -927,6 +932,47 @@ export class NeynarV2APIClient {
     return response.data;
   }
 
+/**
+ * Retrieves detailed information about a cast conversation based on a specified hash or URL. Useful
+ * for fetching in-depth details of a single cast conversation, including replies up to a specified depth. The method
+ * allows for specifying the type of the provided identifier (e.g., a hash or URL) and how deep the reply chain should
+ * be fetched.
+ *
+ * @param {string} castHashOrUrl - The hash or URL of the cast conversation to be retrieved.
+ * @param {CastParamType} type - Specifies the type of the provided identifier, indicating whether it's a hash or URL.
+ * @param {number} [replyDepth] - Optional parameter to specify how deep the reply chain should be fetched.
+ *
+ * @returns {Promise<ConversationContainer>} A promise that resolves to a `ConversationContainer` object,
+ *   containing detailed information about the requested cast conversation, including replies up to the specified depth.
+ *
+ * @example
+ * // Fetch detailed information about a cast conversation using a given hash or URL, with a reply depth of 2
+ * client.lookupCastConversation(
```

Comment:
Add an import statement as well so users can simply copy-paste it

import { CastParamType } from "@neynar/nodejs-sdk";